### ISSUE TITLE
Piechart with two or more metrics

### ISSUE THREAD

**Issue by orwant** [google/google-visualization-issues#1316]
```
What would you like to see us add to this API?

Feature to render a pie chart for a single data table with two or more metrics.
It will be cool if it is possible to select metric also similar to  dimension which
is now available currently.

json data being something like this : 
{
    "cols": [
        {
            "name": "7DayTotals",
            "columnType": "DIMENSION",
            "dataType": "STRING"
        },
        {
            "name": "views",
            "columnType": "METRIC",
            "dataType": "INTEGER"
        },
        {
            "name": "likes",
            "columnType": "METRIC",
            "dataType": "INTEGER"
        },
        {
            "name": "dislikes",
            "columnType": "METRIC",
            "dataType": "INTEGER"
        }
    ],
    "rows": [
        [
            "2013-09-08",
            40205977,
            479061,
            39806
        ],
        [
            "2013-09-06",
            38386819,
            451390,
            36071
        ]
    ]
}


What component is this issue related to (PieChart, LineChart, DataTable,
Query, etc)?

PieChart,DataTable


*********************************************************
For developers viewing this issue: please click the 'star' icon to be
notified of future changes, and to let us know how many of you are
interested in seeing it resolved.
*********************************************************

```

Original issue reported on code.google.com by `chakri@trisys.in` on 2013-09-18 11:28:40

**Comment by orwant** [comment]
```
How do you see this being displayed on a PieChart? Are you imagining a Sunburst chart?
```

Original issue reported on code.google.com by `grabks@google.com` on 2013-09-18 14:24:17

**Comment by orwant** [comment]
```
No...
What I meant was 
We have options for metrics similar to dimensions
say we have views and likes as metrics in datatable(all the metrics are shown in line
,bar chart etc.)
If there were two options views and likes similar to dimensions but for metrics.
Once the chart is loaded it is for views by default and when likes is selected the
 pie chart should change to that of likes and dimensions.

Basically ,what I am trying to say is we should be able to toggle between likes and
views ( which are metrics ) and the pie chart's data should change accordingly. 
```

Original issue reported on code.google.com by `chakri@trisys.in` on 2013-09-18 14:51:53

**Comment by orwant** [comment]
```
It sounds to me like this is a job for controls. We actually have a control in progress
for selecting which column should be used, but it's not quite ready yet. We will ping
this bug when it is released.
```

Original issue reported on code.google.com by `grabks@google.com` on 2013-09-18 15:19:27

**Comment by orwant** [comment]
```
And it would also be cool to have support for sunburst charts... :-)
```

Original issue reported on code.google.com by `moilejter` on 2013-10-29 15:26:40
